packages/mask/src/components/InjectedComponents/ProfileTabContent.tsx

```diff
@@ -46,11 +47,30 @@ export function ProfileTabContent(props: ProfileTabContentProps) {
     const [selectedTab, setSelectedTab] = useState<Plugin.SNSAdaptor.ProfileTab | undefined>()
 
     const identity = useCurrentVisitingIdentity()
-    const { value: addressNames, loading: loadingAddressNames } = useAddressNames(identity)
+    const { value: addressNames = [], loading: loadingAddressNames } = useAddressNames(identity)
 
+    const { loading: loadingTwitterAddressName, value: twitterAddressName } = useNFTContainerAtTwitter(
```

Comment:
Not a separated hook. Plz make it integrate into `packages/mask/src/plugins/Wallet/services/domain.ts`.